tests/unit_tests/utils/date_parser_tests.py

```diff
@@ -290,236 +226,49 @@ def test_datetime_eval() -> None:
     expected = datetime(2018, 1, 1)
     assert result == expected
 
-    result = datetime_eval("datetime('2018-9')")
-    expected = datetime(2018, 9, 1)
-    assert result == expected
-
-    # Parse compact arguments spelling
-    result = datetime_eval("dateadd(datetime('today'),1,year,)")
-    expected = datetime(2017, 11, 7)
-    assert result == expected
-
-    result = datetime_eval("dateadd(datetime('today'), -2, year)")
-    expected = datetime(2014, 11, 7)
-    assert result == expected
```

Comment:
I think you should understand what the previous logic did and then remove the unit test again.